content/en/docs/reference/command-line-tools-reference/feature-gates/DeclarativeValidation.md

```diff
@@ -0,0 +1,16 @@
+---
+title: DeclarativeValidation
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.33"
+---
+Enables running declarative validation of APIs, where declared. When enabled, APIs with
+declarative validation rules will validate objects using the generated
+declarative validation code and compare the results to the regular imperative validation.
+See DeclarativeValidationTakeover for more.
```

Comment:
> `See DeclarativeValidationTakeover for more.`

Change this to a hyperlink to the documentation (which you, or a colleague from the SIG, will need to write).

Reply:
I am happy to write this documentation but I am currently a bit confused as I was intending for this to essentially just link to the other feature gate documentation that is already a part of this PR.  I have added such a link now:

> The original hand-written validation are still the authoritative validations when this is enabled but this can be changed if the [DeclarativeValidationTakeover feature gate](/docs/reference/command-line-tools-reference/feature-gates/DeclarativeValidationTakeover.md) is enabled in addition to this gate.  ...

Can you clarify what we hope for this additional documentation to have?  Thanks!